packages-content-model/roosterjs-content-model-editor/lib/modelApi/edit/utils/deleteSegment.ts

```diff
@@ -26,11 +26,12 @@ export function deleteSegment(
 
     switch (segmentToDelete.segmentType) {
         case 'Br':
-        case 'Image':
         case 'SelectionMarker':
             segments.splice(index, 1);
             return true;
-
+        case 'Image':
+            segments.splice(index, segmentToDelete.link ? 2 : 1);
```

Comment:
What are the 2 segments when delete?

Reply:
The selection marker and the image.

Reply:
So that means selection marker will be removed from model? how does browser know where to focus after this?

Reply:
I feel this issue will happen to all kinds of segments, but not only image.

Maybe we just need to remove the link decorator (as well as code decorator) in `normalizeContentModel` if a selection marker has link and its sibling segments have not.